### PR TITLE
EM - Real Application Testing labs - WMS ID: 7181 

### PR DESCRIPTION
**Pull request by dsuherma**

### CODE REVIEW

**Review by rfontcha — CHANGES_REQUESTED** [review]
@dsuherma larger manifest.json under **freetier**, **livelabs**, and **main** folders is referencing non-existing MD and folders, and the entry in there for your lab is wrong. Please update accordingly and re-commit.
image

**Comment by dsuherma** [comment]
@rfontcha , I fixed 2 manifest.json in livelabs and freetier remove em monitoring entry. Also , fix file reference to correct md file.

File: enterprise-manageability-library/enterprise-manager/workshops/livelabs/manifest.json [hunk 1]
```diff
@@ -54,6 +54,12 @@
       "publisheddate": "6/15/2021",
       "filename": "./../../em-job-system-automation/em-job-system-automation.md"
     },
+    {
+      "title": "Lab 9: EM Real Application Testing Overview",
+      "description": "This lab provides an overview of Configuration and Compliance Management features in . It is intended solely to help you assess the business benefits of upgrading to and to plan your I.T. projects",
```
Comment (rfontcha) [inline enterprise-manageability-library/enterprise-manager/workshops/livelabs/manifest.json:59]: @dsuherma  - Wrong description. Please update and review the same in other manifests for accuracy

**Comment by rfontcha** [comment]
@dsuherma - I assume you submitted a request for new workshop at bit.ly/oraclewms. Include the Workshop ID from from WMS in the title of this pull request

**Comment by rfontcha** [comment]
@dsuherma  - Please update the title for the lab in Manifests and the Lab's MD file first line to "Get Started with Real Application Testing"

**Review by rfontcha — APPROVED** [review]

**Pull request merged**